abis_mapping/templates/survey_site_data_v2/examples/minimal.ttl

```diff
@@ -85,47 +85,47 @@
     tern:locationDescription "Cowaramup Bay Road" .
 
 <http://createme.org/dataset/Example-Systematic-Survey-Site-Dataset> a tern:Dataset ;
-    schema:dateCreated "2024-10-18"^^xsd:date ;
-    schema:dateIssued "2024-10-18"^^xsd:date ;
+    schema:dateCreated "2020-01-01"^^xsd:date ;
+    schema:dateIssued "2020-01-01"^^xsd:date ;
     schema:description "Example Systematic Survey Site Dataset by Gaia Resources" ;
     schema:name "Example Systematic Survey Site Dataset" .
 
 [] a rdf:Statement ;
     geo:hasGeometry [ a geo:Geometry ;
-            geo:asWKT "<http://www.opengis.net/def/crs/EPSG/0/4326> LINESTRING (-33.85 114.99, -33.85 115)"^^geo:wktLiteral ;
+            geo:asWKT "<http://www.opengis.net/def/crs/EPSG/0/4326> POLYGON ((-33.85 114.98, -33.85 115.01, -33.87 115.01, -33.87 114.98, -33.85 114.98))"^^geo:wktLiteral ;
             geo:hasMetricSpatialAccuracy 5e+01 ] ;
-    rdf:object _:N6800c627487c4a8c9a62f59a799545be ;
+    rdf:object _:N53dd57c04e35e4aa3188c0b800000000 ;
     rdf:predicate geo:hasGeometry ;
-    rdf:subject <http://createme.org/dataset/Example-Systematic-Survey-Site-Dataset/Site/P1> ;
+    rdf:subject <http://createme.org/dataset/Example-Systematic-Survey-Site-Dataset/Site/P0> ;
     rdfs:comment "supplied as" .
 
 [] a rdf:Statement ;
     geo:hasGeometry [ a geo:Geometry ;
-            geo:asWKT "<http://www.opengis.net/def/crs/EPSG/0/4326> POLYGON ((-33.85 114.98, -33.85 115.01, -33.87 115.01, -33.87 114.98, -33.85 114.98))"^^geo:wktLiteral ;
+            geo:asWKT "<http://www.opengis.net/def/crs/EPSG/0/4326> LINESTRING (-33.85 114.99, -33.85 115)"^^geo:wktLiteral ;
             geo:hasMetricSpatialAccuracy 5e+01 ] ;
-    rdf:object _:N2952b6066f2b4cebb9b4a50d41ebb8ea ;
+    rdf:object _:N53dd57c04e35e4aa3188c0b800000001 ;
     rdf:predicate geo:hasGeometry ;
-    rdf:subject <http://createme.org/dataset/Example-Systematic-Survey-Site-Dataset/Site/P0> ;
+    rdf:subject <http://createme.org/dataset/Example-Systematic-Survey-Site-Dataset/Site/P1> ;
     rdfs:comment "supplied as" .
 
 [] a rdf:Statement ;
     geo:hasGeometry [ a geo:Geometry ;
             geo:asWKT "<http://www.opengis.net/def/crs/EPSG/0/4326> POINT (-33.85 114.99)"^^geo:wktLiteral ;
             geo:hasMetricSpatialAccuracy 5e+01 ] ;
-    rdf:object _:N1c3ebed8bb624c9ba69063ad0c274e88 ;
+    rdf:object _:N2d5e0873e8e72e68e4a9793300000000 ;
     rdf:predicate geo:hasGeometry ;
     rdf:subject <http://createme.org/dataset/Example-Systematic-Survey-Site-Dataset/Site/P1> ;
     rdfs:comment "supplied as" .
 
-_:N1c3ebed8bb624c9ba69063ad0c274e88 a geo:Geometry ;
+_:N2d5e0873e8e72e68e4a9793300000000 a geo:Geometry ;
     geo:asWKT "<http://www.opengis.net/def/crs/EPSG/0/7844> POINT (-33.85 114.99)"^^geo:wktLiteral ;
     geo:hasMetricSpatialAccuracy 5e+01 .
 
-_:N2952b6066f2b4cebb9b4a50d41ebb8ea a geo:Geometry ;
+_:N53dd57c04e35e4aa3188c0b800000000 a geo:Geometry ;
```

Comment:
see my comment surrounding the searching for the correct frame.  I don't believe these IDs should be different following the N prefix up until their last 1 or 2 digits.

Reply:
The first ID is for a Bnode created in `add_point_geometry`,
the second is for one created in `add_footprint_geometry`,
those functions names are included in the hash, so I would expect these IDs to be different in the hash part of the ID